install/windows.md

```diff
@@ -609,7 +610,7 @@ das machen wir in der Datei `~/.bashrc`.
 Führe dazu folgenden Befehl im Terminal aus:
 
 ```
-echo 'export PATH="$HOME/.local/texlive/2022/bin/x86_64-linux:$PATH"' >> ~/.bashrc
+echo 'export PATH="$HOME/.local/texlive/2023/bin/x86_64-linux:$PATH"' >> ~/.bashrc
```

Comment:
Add $

Reply:
Are you planning on pushing these changes soon?
As far as I know, there is no release of the 2023 version until 19mar: public release (also of MacTeX).
I hope this doesn't confuse people or leads to broken installations due to the bashrc not working properly.

Same goes for Anaconda in this regard.

Reply:
We should not change the years here until the actual release was made.